.pipelines/templates/template-setup-golang-env.yml

```diff
@@ -5,10 +5,11 @@ parameters:
   modulePath: ""
 steps:
   - script: |
+      set -e
       mkdir -p '${{ parameters.gobin }}'
       mkdir -p '${{ parameters.gopath }}/pkg'
       mkdir -p '${{ parameters.modulePath }}'
-      ls -a | grep -v ${{ parameters.gopath }} | xargs mv -t ${{ parameters.modulePath }}
+      ls -a | grep -v `basename ${{ parameters.gopath }}` | grep -v '^\.\.\?$' | xargs mv -t ${{ parameters.modulePath }}
```

Comment:
This seems worse still.  Do either of the following work and are they an improvement?

(remove the initial 2 mkdirs, go tooling should do that for us)
```
mkdir -p '${{ parameters.modulePath }}'
shopt -s dotglob; mv * '${{ parameters.modulePath }}'
```

(remove the initial 2 mkdirs, go tooling should do that for us)
```
mkdir -p '${{ parameters.modulePath }}'
mv * .git '${{ parameters.modulePath }}'
```

Reply:
I went with the shopt path because we need the .sha256sum file in the CI pipeline (and maybe other dotfiles). I turned off -e for the mv to ignore the failed attempt to copy the gopath folder into itself.